components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -1162,6 +1257,40 @@ void triggerPreRevokeListeners(OAuthRevocationRequestDTO
         }
     }
 
+    void triggerPreApplicationTokenRevokeListeners(OAuthAppRevocationRequestDTO revokeRequestDTO,
```

Comment:
shouldn't the method name be changed as `triggerPostApplicationTokenRevokeListeners`?

Reply:
Addressed in [6918b82](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1627/commits/6918b828f239ba766e20f66e6d0af997d891413a)